### ISSUE TITLE
[PRE REVIEW]: Habfuzz: A Fortran tool to calculate the instream hydraulic habitat suitability based on fuzzy logic

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#77]
**Submitting author:** @chtheodoro (<a href="http://orcid.org/0000-0002-5395-0347">Christos Theodoropoulos</a>)
**Repository:** <a href="https://github.com/chtheodoro/Habfuzz" target ="_blank">https://github.com/chtheodoro/Habfuzz</a>
**Version:** v1.0.0
**Editor:** @labarba
**Reviewer:** @fonnesbeck

**What this issue is for**

Thanks for submitting your paper to JOSS @chtheodoro. The JOSS editor (shown at the top of this issue) will work with you on this issue to find a reviewer for your submission before creating the main review issue.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon. I'm here to help you with some common editorial tasks for JOSS. @labarba it looks like you're currently assigned as the editor for this paper :tada:

For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by arfon** [comment]
@labarba - does this look like a paper you think you might be able to handle?

**Comment by chtheodoro** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List all of Whedon's capabilities
@whedon commands

# Assign a GitHub user as the reviewer of this submission
@whedon assign @username as reviewer

# List the GitHub usernames of the JOSS editors
@whedon list editors

# List of JOSS reviewers together with programming language preferences and domain expertise
@whedon list reviewers

# Change editorial assignment
@whedon assign @username as editor

# Set the software archive DOI at the top of the issue e.g.
@whedon set 10.0000/zenodo.00000 as archive

# Open the review issue
@whedon start review
```

:construction: **Important** :construction:

This is all quite new. Please make sure you check the top of the issue after running a @whedon command (you might also need to refresh the page to see the issue update).

**Comment by chtheodoro** [comment]
@whedon list reviewers

**Comment by whedon** [comment]
Here's the current list of JOSS reviewers: https://gist.github.com/arfon/5317c568cb32c7b917fea3c13958131d

**Comment by labarba** [comment]
@fonnesbeck — I'm a bit at a loss here to shepherd the review of this paper, as I'm unfamiliar with the methods and application mentioned in the description ... I remember that you worked in some ecological applications of stats and data science: would you be able to review this paper for JOSS, or to suggest another appropriate reviewer?
Reviewer guidelines at:
http://joss.theoj.org/about#reviewer_guidelines

**Comment by chtheodoro** [comment]
If I am allowed to propose, maybe @jsta could help here as I see he has a strong ecological background. Although we need someone familiar with fuzzy logic, the submission is not hard to follow if you see the README.md. I am here also for any questions 😊

**Comment by fonnesbeck** [comment]
@labarba I will have a look, and either review it myself or suggest another. I have done some stream ecology work in the past, but not these methods.

**Comment by fonnesbeck** [comment]
I will volunteer to review the paper.

**Comment by labarba** [comment]
@whedon assign @fonnesbeck as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @fonnesbeck

**Comment by arfon** [comment]
@whedon start review

**Comment by whedon** [comment]
You didn't say the magic word! Try this:

```
@whedon start review magic-word=bananas
```

**Comment by arfon** [comment]
@whedon start review magic-word=bananas

**Comment by arfon** [comment]
OK, the review is issue is over here: https://github.com/openjournals/joss-reviews/issues/82

@fonnesbeck - please take a look at that issue for completing the review.
